### PR TITLE
Development

### PR DESCRIPTION
**Pull request by RyanPnayR**

### CODE REVIEW

**Comment by benjamingregory** [comment]
@aloisbarreras This looks fine to me but you're the better judge. Can you make sure it passes the new testing procedure you set up and merge it in?

File: README.md [hunk 1]
```diff
@@ -188,42 +186,81 @@ An Aries integration to query data from SurveyMonkey's api.
 `getErrorById` - Returns an error.
 `params` : errorId
 
-## Configuration
+##Configuration
 
-### apiToken
-This is your SurveyMonkey app's api key. Located in settings/credentials under the MyApps Tab. 
+###ExampleConfig
```
Comment (trong-le) [inline README.md]: Rename to ```Example Configuration```

File: README.md [hunk 2]
```diff
@@ -188,42 +186,81 @@ An Aries integration to query data from SurveyMonkey's api.
...
-apiToken: "secret",
+{
+        authToken: process.env.SurveyMonkeyAUTHTOKEN,
+        version:"v3",
+        method:"getSurveys"
+}
 ```
 
-### authToken
+###apiToken
```
Comment (trong-le) [inline README.md]: Rename to API Token

File: README.md [hunk 3]
```diff
@@ -188,42 +186,81 @@ An Aries integration to query data from SurveyMonkey's api.
...
+This is your SurveyMonkey app's api key. Located in settings/credentials under the MyApps Tab. 
+```javascript
+{
+    apiToken: "secret",
+}      
+```
+**Note: This is not required always. Depends on app. Default should be left blank
+###authToken
```
Comment (trong-le) [inline README.md]: Rename to Auth Token

File: README.md [hunk 4]
```diff
@@ -188,42 +186,81 @@ An Aries integration to query data from SurveyMonkey's api.
...
 This is your SurveyMonkey app's access token. Located in settings/credentials under the MyApps Tab. 
 ```javascript
-authToken: "super secret",
+{
+    authToken: "super secret",
+}
 ```
 
-### version
+###version
```
Comment (trong-le) [inline README.md]: Capital Version

File: README.md [hunk 5]
```diff
@@ -188,42 +186,81 @@ An Aries integration to query data from SurveyMonkey's api.
...
 This is the version of the api you want to use. 
 ```javascript
-version: "v3",
+{
+    version: "v3",
+}
 ```
 
-### method
+###method
```
Comment (trong-le) [inline README.md]: Capital Method

File: README.md [hunk 6]
```diff
@@ -188,42 +186,81 @@ An Aries integration to query data from SurveyMonkey's api.
...
 This is the method in the program you want to call.
- ```javascript
-method: "getSurveys",
+```javascript
+{
+    method: "getSurveys",
+}
 ```
+    
+###ExampleReturn
```
Comment (trong-le) [inline README.md]: Change to ```Response``` then have a description underneath that tells which method this is an example response for.

**Comment by trong-le** [comment]
@RyanPnayR Add LICENSE file, rename DOCKERFILE to Dockerfile, add scripts to circle.yml flle. Also, in functions that use the config but don't pass it through, change parameter in function to be functionName({ a, b, c, etc. }) rather than just calling config.a, config.b, config.c, config.etc every time

**Comment by trong-le** [comment]
@RyanPnayR have you read my comments/made changes?

**Comment by benjamingregory** [comment]
@RyanPnayR Please confirm you've seen @trong-le's previous comments and provide a status update on this.

**Pull request merged**